nx-dev/nx-dev/next.config.js

```diff
@@ -190,6 +190,14 @@ module.exports = withNx({
         permanent: true,
       });
     }
+    // Nesting 5min tutorials in Getting Started
+    for (let s of Object.keys(redirectRules.nested5minuteTutorialUrls)) {
```

Comment:
this one works now, but I think we should check whether we didn't miss any other in there.
I didn't find these two mapped for instance:

https://github.com/nrwl/nx/blob/b1a9ae7ce76697019b26ec9a172005840cca8da6/nx-dev/nx-dev/redirect-rules.js#L696-L697

Also, for the `extraNodeRedirects` the destination of the first entry is wrong. It should be pointing to `/getting-started/tutorials/node-server-tutorial`

Reply:
Fixed the `extraNodeRedirects`. Let's tackle the rest in a followup PR.